ups/sphgeom.table

```diff
@@ -1,2 +1,7 @@
+setupRequired(base)
+setupRequired(sconsUtils)
+setupOptional(doxygen)
+
 envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
 envPrepend(DYLD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
```

Comment:
Not objecting, but why? What is LSST_LIBRARY_PATH?

Also, should I make the same change in ups/TMPL.table in the templates package?